src/config.rs

```diff
@@ -1373,6 +1373,11 @@ impl Config {
         self.dynamic_title
     }
 
+    #[inline]
+    pub fn dynamic_title_mut(&mut self) -> &mut bool {
+        &mut self.dynamic_title
+    }
```

Comment:
`dynamic_title` is just a boolean, so I don't think it makes much sense to return a mutable reference. It seems more logical to add a direct "setter" that would just take `(&mut self, new: bool)` and mutate the value internally.

It might even be a good idea to make this even more specific and just call it `disable_dynamic_title`. So it can't be used to enable `dynamic_title` unless the code is changed again, which would make it the minimum amount of mutability necessary for it to work properly.

Reply:
I'm ok with that, I'm just going on conventions I'm used to lol.